specification/schemas/payment-providers/PaymentProviderResource.json

```
@@ -0,0 +1,19 @@
+{
+  "type": "object",
+  "required": [
+    "type"
+  ],
+  "properties": {
+    "type": {
+      "type": "string",
+      "pattern": "^payment-providers$",
+      "example": "payment-providers"
+    },
+    "id": {
+      "type": "string",
+      "format": "uuid",
+      "pattern": "^[a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12}$",
+      "example": "7fac478b-b3c4-46e6-b29e-a31599d27a7a"
```

This UUID is already used for our Return resource. Let's generate a fresh one for this resource.

```suggestion
      "example": "6718621d-521c-4045-b4ef-1ab4dc29b316"
```